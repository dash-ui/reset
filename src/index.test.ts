import reset from './index'

describe('reset', () => {
  it('is what it is', () => {
    expect(reset).toMatchSnapshot()
  })
})
