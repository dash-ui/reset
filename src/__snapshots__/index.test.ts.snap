// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reset is what it is 1`] = `"*,*::before,*::after{box-sizing:border-box;}*{margin:0;}html{word-wrap:break-word;text-size-adjust:100%;}body{scroll-behavior:smooth;}ul,ol{padding:0;list-style:none;}nav{border:0;}video,img{max-width:100%;display:block;}input,button,textarea,select{font:inherit;}@media (prefers-reduced-motion:reduce){*{animation-duration:0.01ms!important;animation-iteration-count:1!important;transition-duration:0.01ms!important;scroll-behavior:auto!important;}}"`;
